rp2040-hal-macros/Cargo.toml

```diff
@@ -6,6 +6,8 @@ readme = "README.md"
 version = "0.1.0"
 edition = "2021"
 repository = "https://github.com/rp-rs/rp-hal"
+categories = ["embedded", "hardware-support", "no-std", "no-std::no-alloc"]
+keywords = ["embedded", "hal", "raspberry-pi", "rp2040", "embedded-hal"]
```

Comment:
Are these actually useful? I don't think someone searching for one of these keywords would be interested in rp2040-hal-macros.

Reply:
i think it might help depending on what people search for. e.g. ["embedded rp2040"](https://crates.io/search?q=embedded+rp2040) only yields `rp2040-hal` as the 5th result - though i of course have no numbers for how often people search things like this. i think the categories are more important, but the keywords certainly don't hurt

Reply:
This comment is only about the macros crate. Finding `rp2040-hal` easily is of course useful!
But who needs `rp2040-hal-macros`, if not as a dependency of `rp2040-hal`?

Reply:
that's true. i just thought "well, that can't hurt". i can also remove it here if you prefer

Reply:
ok, done